src/app/components/shared/country-picker/CountryModal.js

```diff
@@ -1,54 +1,71 @@
 import React, { Component } from 'react';
+import { injectIntl, intlShape } from 'react-intl';
 import PropTypes from 'prop-types';
+import countries from 'i18n-iso-countries';
+import bokmalContriesList from 'i18n-iso-countries/langs/nb.json';
+import nynorskContriesList from 'i18n-iso-countries/langs/nn.json';
+
 import { Select } from 'nav-frontend-skjema';
 import { Undertittel, Element } from 'nav-frontend-typografi';
 import Modal from 'nav-frontend-modal';
-import DateInput from 'shared/date-input/DateInput';
 import { Knapp } from 'nav-frontend-knapper';
 
-const selectOptions = ['Marshalløyene', 'Argentina', 'Kina'];
-
-const renderSelectOptions = (selectOption) =>
-	selectOption.map((optionValue) => (
-		<option key={optionValue} value={optionValue}>
-			{optionValue}
-		</option>
-	));
+import DateInput from 'shared/date-input/DateInput';
+import getMessage from '../../../util/i18n/index';
 
 class CountryModal extends Component {
 	constructor(props) {
 		super(props);
+		const contriesLanguage =
+			props.language === 'nb' ? bokmalContriesList : nynorskContriesList;
+		countries.registerLocale(contriesLanguage);
+		const { intl } = props;
 		if (props.visit) {
 			this.state = {
-				titleText: 'Endre utenlandsopphold',
-				submitButtonText: 'Lagre endringer',
+				titleText: getMessage(intl, 'medlemmskap.landvelger.endre'),
+				submitButtonText: getMessage(intl, 'medlemmskap.landvelger.lagre'),
 				...props.visit
 			};
 		} else {
 			this.state = {
-				titleText: 'Legg til utenlandsopphold',
-				submitButtonText: 'Legg til land',
-				country: '',
-				startDate: '',
-				endDate: ''
+				titleText: getMessage(intl, 'medlemmskap.landvelger.leggTil'),
+				submitButtonText: getMessage(
+					intl,
+					'medlemmskap.landvelger.leggTilLand'
+				),
+				land: '',
```

Comment:
Trenger du å spesifisere tomme strenger for alle feltene her, hvis disse allerede er satt som defaultProps på visit?